src/components/common/App/AppInput.vue

```diff
@@ -0,0 +1,36 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+  props: {
+    placeholder: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      inputContent: '',
```

Comment:
And likewise with this one - I'd recommend introducing a `v-model` implementation here too 😄
I feel like you'll handle this just fine but let me know if you need any help with this one 💪